frontend/pkg/frontend/ocm.go

```diff
@@ -52,6 +52,8 @@ const (
 	csOutboundType                      string = "load_balancer"
 	csUsernameClaimPrefixPolicyNoPrefix string = "NoPrefix"
 	csUsernameClaimPrefixPolicyPrefix   string = "Prefix"
+	csCIDRBlockAllowAccessModeAllowAll  string = "allow_all"
```

Comment:
I don't see `AllowAll` being used.  Is this implied when setting `API.CIDRBlockAccess(nil)`?

Reply:
Pushed the change to use `AllowAll` in case of nil/empty array.